test/end-to-end/test_studio_supervisor.ps1

```diff
@@ -37,7 +27,7 @@ Describe "Studio supervisor" {
 
     AfterAll {
         if($env:DOCKER_STUDIO_TEST) {
-            docker ps -q | ForEach-Object { docker stop $_ }
+            docker ps -q --filter "ancestor=habitat/win-studio-x86_64-windows:ltsc2019-$(($cliVersion -split '/')[0])" | ForEach-Object { docker stop $_ }
```

Comment:
Definitely an improvement over before.  This reminded me that we are on a shared host (I think), so docker stop of everything may have been catching containers that were not ours.  This _should_ be safe from that, as at this point we should be the only ones running this container version.   Ad-hoc runs of this pipeline should be safe as well, as those are typically done off of feature branches which will have an unreleased version.

Reply:
For example, this is what the automate team does: https://github.com/chef/automate/blob/master/.buildkite/hooks/pre-exit#L10

Reply:
I made some changes to take a similar approach.